### DISCUSSION TITLE
About the flexibility of the relationship

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#2041] [Feature Requests & Ideas]
Let's start with the example in the [relationship document](https://payloadcms.com/docs/fields/relationship)
<img width="882" alt="image" src="https://user-images.githubusercontent.com/16400882/217543881-64a2ea55-9213-48b1-bf53-6bd645f8652f.png">

By default, ObjectID is used as the value of relationship, but if I want to use other fields of the owner collection, it is very inconvenient. 
Suppose the owner collection has other fields such as name, desc. How can I get the name and desc conveniently?

For example, through the REST API mentioned in the documentation.

I feel the above process is too cumbersome. Is there an easy way?

**Comment by blueMountain007** [comment]
@jmikrut  Do you have any ideas? Thanks

**Comment by DanRibbens** [comment]
Are you asking how to query a collection on the fields of a relationship?

You can do that with dot notation:

`/api/cars?where[owner.name][equals]=Dan`

If that isn't what you're after, can you explain in more detail what is you're trying to do?

Thanks!
